src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -41,19 +44,23 @@ public class BlockStoreSignaled : SignalObserver<Block>
         /// <summary>Queue of chained blocks that will be announced to the peers.</summary>
         private readonly ConcurrentQueue<ChainedBlock> blocksToAnnounce;
 
+        /// <summary>Event slim that is set when a new block gets enqueued.</summary>
```

Comment:
remove slim from comment, that is an implementation detail

Reply:
Fixed.